### ISSUE TITLE
Code Release?

### ISSUE THREAD

**Issue by WinKawaks** [rtous/lester#1]
Very cool work! Will the code release soon?

**Comment by rtous** [comment]
Yes I will release the code but I have to refactor it a little and it will take a few weeks.
